ext/sst/sst_responder_subcomponent.hh

```diff
@@ -45,19 +45,26 @@
 // from gem5
 #include <sim/sim_object.hh>
 #include <sst/outgoing_request_bridge.hh>
+#include <sst/external_memory.hh>
 #include <sim/root.hh>
 #include <sst/sst_responder_interface.hh>
+#include <mem/backdoor.hh>
 
 #include "translator.hh"
 #include "sst_responder.hh"
 
 class SSTResponderSubComponent: public SST::SubComponent
 {
   private:
-    gem5::OutgoingRequestBridge* responseReceiver;
+    // gem5::OutgoingRequestBridge* responseReceiver;
+    // responseReceiver for this branch is hardcoded to ExternalMemory*.
+    // TODO: We need to make a better design to handle multiple types of
+    // outgoing request classes.
```

Comment:
This must be done sooner rather than later. We can chat about the best approach.

Reply:
I agree. I ran into this issue yesterday so I tried to solve it hastily.